server/websockets.ts

```diff
@@ -30,8 +30,10 @@ const setupWebSocketServer = () => {
 };
 
 function handlerSubscribeStudent(parsed: StudentSubPayload, ws: WebSocketClient) {
-    const selectedLecture: Lecture | undefined = [...lectures.values()].find((lecture) => lecture.link === parsed.data.lecture_link);
+    const selectedLecture: Lecture | undefined = linkLectureMap.get(parsed.data.lecture_link);
     const selectedStudent: Student | undefined = selectedLecture?.studentList.getStudent(parsed.data.student_id);
+    console.log(parsed.data)

```

Comment:
I reckon that this function is going to be removed in the future as it is used only for testing

Reply:
Good point it is unnecessary now.